scripts/sync.py

```diff
@@ -158,6 +159,9 @@ def do(args, Lochness):
             for Module in args.source:
                 lochness.attempt(Module.sync, Lochness, subject, dry=args.dry)
 
+    # annonymize PII
+    dpanonymize.lock_lochness(Lochness)
+
```

Comment:
Current master already has these lines:

![image](https://user-images.githubusercontent.com/35086881/126003921-b3874419-dc9f-4060-93c6-d42e529526b0.png)


I wonder why GitHub still shows them in diff.

Reply:
Not so sure. I must have done something wrong on the way.